interface/batchcom/batchcom.inc.php

```diff
@@ -3,21 +3,21 @@
 //validation functions
```

Comment:
On this script can fix sql injection vulnerability with:
``````

1
--
function register_email($patient_id, $sent_by, $msg_type, $msg_subject, $msg_text) {
    
   $sql="INSERT INTO batchcom SET ";
   $sql.="patient_id=?, ";
   $sql.="sent_by=?, ";
   $sql.="msg_type=?, ";
   $sql.="msg_subject=?, ";
   $sql.="msg_text=?, ";
   $sql.="msg_date_sent=NOW() ";
    
   echo $sql;
    
   $res = sqlStatement($sql, array($patient_id, $sent_by, $msg_type, $msg_subject, $msg_text));
   }
``````